app/helpers/view_helper.rb

```diff
@@ -61,11 +61,23 @@ def enrichment_error_url(provider_code:, course:, field:, message: nil)
         required_qualifications: "#{base}/requirements?display_errors=true#required_qualifications_wrapper",
         age_range_in_years: "#{base}/age-range?display_errors=true",
         sites: "#{base}/schools?display_errors=true",
-        study_sites: (course.provider&.study_sites&.none? ? "#{provider_base}/study-sites" : "#{base}/study-sites").to_s
+        study_sites: (course.provider&.study_sites&.none? ? "#{provider_base}/study-sites" : "#{base}/study-sites").to_s,
+        accrediting_provider: accrediting_provider_error_links(message, course)
```

Comment:
Spotted a quirk with this where if there are no accredited providers under the provider, the validation error takes you to the update page. There is no journey for this currently as there is nothing for the user to choose. The error should take you to the page to add a new accredited provider and the error message wording should reflect that too.

I think `study_sites` above is doing something similar, I know the whole method could do with a refactor but maybe we could replicate what we've done there for now to help with readability of the method as a whole?

![validation link should be dynamiic](https://github.com/DFE-Digital/publish-teacher-training/assets/50492247/a8ad3606-315e-4a96-a8e4-750db61b40cd)

Reply:
The content in the column next to 'Accredited provider' should read 'Select an accredited provider'.